homeassistant/components/electrolux_group/sensor.py

```diff
@@ -0,0 +1,336 @@
+
+_LOGGER = logging.getLogger(__name__)
+
+ELECTROLUX_TO_HA_TEMPERATURE_UNIT = {
+    "CELSIUS": UnitOfTemperature.CELSIUS,
+    "FAHRENHEIT": UnitOfTemperature.FAHRENHEIT,
+}
+
+
+@dataclass(frozen=True)
+class ElectroluxSensorDescription(SensorEntityDescription):
+    """Custom sensor description for Electrolux sensors."""
+
+    name: str = ""
+    value_fn: Callable[..., Any] = lambda *args: None
+    is_supported_fn: Callable[..., Any] = lambda *args: None
+
+
+GENERAL_ELECTROLUX_SENSORS: tuple[ElectroluxSensorDescription, ...] = (
+    ElectroluxSensorDescription(
+        key="connection_state",
+        name="Connection state",
+        icon="mdi:wifi",
+        value_fn=lambda appliance: appliance.state.connectionState,
+    ),
+)
+
+OVEN_ELECTROLUX_SENSORS: tuple[ElectroluxSensorDescription, ...] = (
+    ElectroluxSensorDescription(
+        key="start_at",
+        name="Start at",
+        icon="mdi:timer-play-outline",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        value_fn=lambda appliance: appliance.get_current_start_at(),
+        is_supported_fn=lambda appliance: appliance.is_feature_supported(START_TIME),
+    ),
+    ElectroluxSensorDescription(
+        key="runnig_time",
+        name="Running time",
+        icon="mdi:timer-outline",
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        suggested_unit_of_measurement=UnitOfTime.HOURS,
+        state_class="measurement",
+        value_fn=lambda appliance: appliance.get_current_running_time(),
+        is_supported_fn=lambda appliance: appliance.is_feature_supported(RUNNING_TIME),
+    ),
+    ElectroluxSensorDescription(
+        key="appliance_state",
+        name="Appliance state",
+        icon="mdi:information-outline",
+        value_fn=lambda appliance: appliance.get_current_appliance_state(),
+        is_supported_fn=lambda appliance: appliance.is_feature_supported(
+            APPLIANCE_STATE
+        ),
+    ),
+    ElectroluxSensorDescription(
+        key="food_probe_state",
+        name="Food probe state",
+        icon="mdi:thermometer-probe",
+        value_fn=lambda appliance: appliance.get_current_food_probe_insertion_state(),
+        is_supported_fn=lambda appliance: appliance.is_feature_supported(
+            FOOD_PROBE_STATE
+        ),
+    ),
+    ElectroluxSensorDescription(
+        key="door_state",
+        name="Door state",
+        icon="mdi:door",
+        value_fn=lambda appliance: appliance.get_current_door_state(),
+        is_supported_fn=lambda appliance: appliance.is_feature_supported(DOOR_STATE),
+    ),
+    ElectroluxSensorDescription(
+        key="remote_control",
+        name="Remote control",
+        icon="mdi:remote",
+        value_fn=lambda appliance: appliance.get_current_remote_control(),
+        is_supported_fn=lambda appliance: appliance.is_feature_supported(
+            REMOTE_CONTROL
+        ),
+    ),
```

Comment:
What's the state of these 2? They look like on/off, which would make them a binary sensor

Reply:
door state should be a binary sensor then

What does `NOT_SAFETY_RELEVANT_ENABLED` mean?